### DISCUSSION TITLE
Old MARTINI Forum

### DISCUSSION THREAD

**Discussion by patrickcorrigan54** [Martini-Force-Field-Initiative/MartiniDiscussions#4] [General]
Hello, I'm currently trying to troubleshoot a MARTINI 2 simulation, and there are posts on the old MARTINI forum that address my issues. However, when I try to access these posts, the new MARTINI website tells me "Page Not Found" and I can't figure out how to access the legacy forum. Is there any way I can access the old forum? It is really really useful.

**Comment by csbrasnett** [comment]
Hi @patrickcorrigan54, sorry about this. The forum's been archived and we're currently working towards making the old posts accessible again. In the meantime, we can try and help with your problem here?

**Reply by patrickcorrigan54** [reply]
Thanks for the response! As for my issue, I am simulating a 15 nucleotide dsDNA molecule with a custom/new ligand covalently attached in Martini 2. When I simulate this molecule in a box of solvent, the run will successfully complete the following equilibration scheme:
NVT with strong position restraints for 10 ns
NPT with strong position restraints for 10 ns
NPT with weak position restraints for 10 ns
NPT with no position restraints for 50 ns
But then when it enters the production run (which has the same parameters as the final NPT equilibration run) the system will run for few hundred ns or more but eventually crashes, and gives the error:
Fatal error:
One or more atoms moved too far between two domain decomposition steps.
This usually means that your system is not well equilibrated.
Sometimes it will list specific atoms that moved too far, but they will usually be solvent atoms so Im not sure why they would do that. The fails during production also seem to be random, they happen after a different amount of steps each time. Im not sure how to troubleshoot or fix this - let me know if there is any other information I should provide!

**Reply by csbrasnett** [reply]
I can't say for definite, but a few thoughts from the top of my head -

1) try smaller timestep increments during equilibration, eg. do a multi-stage equilibration of 0.002, 0.005, 0.01 ps timesteps
2) check gromacs versions - at least 2021.3 had issues with energy minimisation that may affect subsequent dynamics
3) check water composition, for martini 2 you may well need some antifreeze water (~10%)
4) have you checked your ligand non-covalently bonded on its own, and is it stable? I know you said the dd error appears to be from the solvent, but this may be worth checking too

good luck!
